hub/core/dataset/dataset.py

```diff
@@ -369,16 +369,20 @@ def delete_tensor(self, name: str, large_ok: bool = False):
                 )
                 return
 
-        delete_tensor(name, self.storage, self.version_state)
+        initial_autoflush = self.storage.autoflush
```

Comment:
Good to put this in a try-finally with `self.storage.autoflush` reset in the finally.

Reply:
We do that in other places too. Should be changed everywhere else?

Reply:
Changed everywhere.